node/cmd/guardiand/adminclient.go

```diff
@@ -68,6 +70,10 @@ func init() {
 	SignExistingVaaCmd.Flags().AddFlagSet(pf)
 	SignExistingVaasFromCSVCmd.Flags().AddFlagSet(pf)
 
+	adminClientSignWormchainAddressFlags := pflag.NewFlagSet("adminClientSignWormchainAddressFlags", pflag.ContinueOnError)
+	unsafeDevnetMode = adminClientSignWormchainAddressFlags.Bool("unsafedevnet", false, "Running in unsafe devnet mode")
```

Comment:
why did this poor flag lose it's camel case? should be `unsafeDevMode` to match node?

Reply:
also nit, but maybe the comment should be "Run in unsafe devnet mode" since it will be the helptext